src/components/marker-table/ContextMenu.js

```diff
@@ -26,7 +26,7 @@ type StateProps = {|
   +markers: Marker[],
   +previewSelection: PreviewSelection,
   +committedRange: StartEndRange,
-  +selectedMarker: IndexIntoMarkers,
+  +selectedMarker: IndexIntoMarkers | null,
```

Comment:
Rather than including null checks everywhere in the methods, I think it would be better to have a second component where this property is required. It doesn't really make sense to even be able to fire those methods without a marker present

```
render() {
  return this.props.selectedMarker ? <MarkerContextMenuContents {...this.props} /> : null;
}
```

Reply:
I tried doing something like this but Flow makes it hard (I think a newer Flow fixes some of these issues):
Cannot create MarkersContextMenuContents element because:
 • inexact props [1] is incompatible with exact ContentsProps [2].
 • null [3] is incompatible with number [4] in property selectedMarker.

```
        src/components/marker-table/ContextMenu.js
    [4]  32│   +selectedMarker: IndexIntoMarkers,
         33│   +updatePreviewSelection: typeof updatePreviewSelection,
         34│ |};
         35│
    [2]  36│ class MarkersContextMenuContents extends PureComponent<ContentsProps> {
           :
    [3] 116│   +selectedMarker: IndexIntoMarkers | null,
           :
        129│     return null;
        130│   }
        131│
    [1] 132│   return <MarkersContextMenuContents {...props} />;
        133│ }
        134│
        135│ const options: ExplicitConnectOptions<{||}, StateProps, DispatchProps> = {
```

So I'll defer this suggestion for now, especially that the `CallNodeContextMenu` uses the same style.